nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -757,6 +757,12 @@ auth required pam_succeed_if.so uid >= 1000 quiet
      </warning>
     </para>
    </listitem>
+   <listitem>
+     <para>
+       TokuDB plugin has been disabled in <package>mariadb</package> 10.5. It is recommended to switch
```

Comment:
```suggestion
       The TokuDB storage engine will be disabled in <package>mariadb</package> 10.5. It is recommended to switch
```

Just to make sure we didn't switch to 10.5 yet, but users are aware to switch to RocksDB.